### ISSUE TITLE
After upgrade files_reader to 1.2.1 the whole files are not shown in brower.

### ISSUE THREAD

**Issue by cdamken** [Yetangitu/owncloud-apps#75]
files_reader 1.2.1
ownCloud 10.0.4
Ubuntu 16.04
Php 7.0

I upgrade from1.1.2 to 1.2.1 today, (I assume it was 1.1.2)

All the files are in the server and you can browse on Android devices (that's the only device I tested), but in the server, when you open the files app all the files are not shown anymore.

In log files I just found this:

``` 
{"reqId":"eg7S5gpoiWZL55FHSB1I","level":1,"time":"2018-01-23T12:19:04+01:00","remoteAddr":"91.45.209.38","user":"usernaMe","app":"files_reader","method":"POST","url":"/index.php/apps/market/apps/files_reader/install","message":"user <carlos>, displayName <Carlos Damken> enabled app "files_reader" [CLIENT_IP: 91.45.209.38] [USER_AGENT: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:57.0) Gecko/20100101 Firefox/57.0]"}
{"reqId":"Z8l1tRt4Q2P1spqRDbKV","level":3,"time":"2018-01-23T12:21:15+01:00","remoteAddr":"91.45.209.38","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=File.PDF","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"Tg3wXWM4Cc1mlap0OKOG","level":3,"time":"2018-01-23T12:22:31+01:00","remoteAddr":"91.45.209.38","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=File.PDF","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"stGgmK5VZIBoK80EbgRG","level":3,"time":"2018-01-24T11:40:09+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=File.PDF","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"FkHOG1XiEvRjhuo76VNZ","level":3,"time":"2018-01-24T11:41:04+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"ggSFr4HOqsz1S1lGFE1I","level":3,"time":"2018-01-24T11:42:03+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"r8R3soAi0q4VMAiBWIT3","level":3,"time":"2018-01-24T11:42:25+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"0bDtUwClmlQSIgqCR5M7","level":3,"time":"2018-01-24T11:44:09+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"HkrF8JtIeglKrAwiUriB","level":3,"time":"2018-01-24T11:44:23+01:00","remoteAddr":"141.75.152.110","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"h5uT06KqKayq0lJn5cBf","level":3,"time":"2018-02-01T16:07:07+01:00","remoteAddr":"217.83.108.99","user":"usernAme","app":"PHP","method":"GET","url":"/index.php/apps/files_reader/?file=FiLe.pdf","message":"Undefined index: title at /var/www/owncloud/apps/files_reader/templates/pdfreader.php#37"}
{"reqId":"x8WJinPUfGsjUK4eXAEa","level":3,"time":"2018-02-01T16:29:38+01:00","remoteAddr":"217.83.108.99","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"4INJKuEnbqL8l5hFrGs2","level":3,"time":"2018-02-01T16:29:42+01:00","remoteAddr":"217.83.108.99","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"H57eAl7v4282oeEE9wMq","level":3,"time":"2018-02-01T16:35:21+01:00","remoteAddr":"217.83.108.99","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"c6eq03HLiwuUT2sFJ0g5","level":3,"time":"2018-02-01T16:42:29+01:00","remoteAddr":"217.83.108.99","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"colCrR31pKCV3GvSOsiB","level":3,"time":"2018-02-01T17:05:41+01:00","remoteAddr":"80.187.102.212","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"WaAl8O9J0AyUCosxHqPE","level":3,"time":"2018-02-01T17:05:46+01:00","remoteAddr":"80.187.102.212","user":"UsernaME","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"YONanpeaIV1eOXcGuxVv","level":3,"time":"2018-02-01T17:06:06+01:00","remoteAddr":"80.187.102.212","user":"UsernaME","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"qd52nN3ysXGNxJC8g0aG","level":3,"time":"2018-02-01T17:06:13+01:00","remoteAddr":"80.187.102.212","user":"UsernaME","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"k7N4lXUysu0rjkj1Yig3","level":3,"time":"2018-02-01T17:06:20+01:00","remoteAddr":"80.187.102.212","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"crHWRDLjJ65B9DmrFABG","level":3,"time":"2018-02-01T19:20:11+01:00","remoteAddr":"189.175.228.129","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"NTdxWHR0NmuknuGiC1Fv","level":3,"time":"2018-02-01T19:20:23+01:00","remoteAddr":"189.175.228.129","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"x5ENGcHk7hOKG3QEigqm","level":3,"time":"2018-02-01T19:20:44+01:00","remoteAddr":"189.175.228.129","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"U9cGc2eNg3fLvKEFlaSX","level":3,"time":"2018-02-01T19:21:00+01:00","remoteAddr":"189.175.228.129","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"x4VPXfPk1tY9sxMEWX92","level":3,"time":"2018-02-01T19:21:09+01:00","remoteAddr":"189.175.228.129","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"LBtG5DxMvTqTx3LIyugs","level":3,"time":"2018-02-01T19:21:17+01:00","remoteAddr":"189.175.228.129","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"WE5zkk5u11ZplOcwYsiT","level":3,"time":"2018-02-01T19:22:01+01:00","remoteAddr":"189.175.228.129","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"LUyrK4nhEmXl20eOEbuL","level":3,"time":"2018-02-01T19:29:42+01:00","remoteAddr":"141.75.149.210","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"00mQYOW6MyLjRkNv6Bgn","level":3,"time":"2018-02-01T19:31:13+01:00","remoteAddr":"189.248.21.202","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"8UH8wJe7EUPs9WckPkHI","level":3,"time":"2018-02-01T19:31:25+01:00","remoteAddr":"189.248.21.202","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Ip8wbu8Wx7gGc5uBlHe3","level":3,"time":"2018-02-01T19:31:37+01:00","remoteAddr":"189.248.21.202","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"wjwzEqoaqiIfDoKmfMHR","level":3,"time":"2018-02-01T20:55:39+01:00","remoteAddr":"131.188.23.98","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"3BZWmnmAM7qLCbMhVlBg","level":3,"time":"2018-02-01T20:55:48+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"e1dwwlJLcLKNTK6Jp4kU","level":3,"time":"2018-02-01T20:56:10+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"bWCB8h18Jut9pU4FP56I","level":3,"time":"2018-02-01T20:56:17+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"cLepLT1X8x9mFOok1zgH","level":3,"time":"2018-02-01T20:56:24+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Kbh5OoTU9ogFnMTzt99E","level":3,"time":"2018-02-01T21:01:55+01:00","remoteAddr":"131.188.23.98","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"1Df24hN0U03v8vUHmZzb","level":3,"time":"2018-02-01T21:02:03+01:00","remoteAddr":"131.188.23.98","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"q2lYmfCiFVAAjG1QBoMz","level":3,"time":"2018-02-01T21:02:07+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Xp8TJdbrzyz8FhDqiDUc","level":3,"time":"2018-02-01T21:02:18+01:00","remoteAddr":"131.188.23.98","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"G0voThw1Hol1IjYvnt9I","level":3,"time":"2018-02-01T21:02:23+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"LHyyQHC8Eve8CuEUYsmE","level":3,"time":"2018-02-01T21:02:26+01:00","remoteAddr":"131.188.23.98","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Vkq4q0MShxihHZicOGL8","level":3,"time":"2018-02-01T21:31:29+01:00","remoteAddr":"177.228.173.217","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"HZqTHldWC8SCa2Lzf8Wa","level":3,"time":"2018-02-01T21:31:40+01:00","remoteAddr":"188.194.5.243","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"vmtueXvsI2Y2xdAFs6MN","level":3,"time":"2018-02-01T21:31:46+01:00","remoteAddr":"188.194.5.243","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"8drrMLxOq9VvazTepbfT","level":3,"time":"2018-02-01T21:31:48+01:00","remoteAddr":"177.228.173.217","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"qZI5XH9cqipaX0OKvypW","level":3,"time":"2018-02-01T21:32:01+01:00","remoteAddr":"188.194.5.243","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"djtK0lJImzRzlnLhxh5o","level":3,"time":"2018-02-01T21:32:51+01:00","remoteAddr":"188.194.5.243","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"iSsMAJ9oVJm8Ek8mkUP8","level":3,"time":"2018-02-01T21:33:25+01:00","remoteAddr":"188.194.5.243","user":"Username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Ix9960R6s46HAYzSoFCe","level":3,"time":"2018-02-01T21:36:52+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"aFx2ERcn2XDCKT8keMaH","level":3,"time":"2018-02-01T21:37:03+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"ZKwQbPJX0zjIQJwq6v8j","level":3,"time":"2018-02-01T21:37:16+01:00","remoteAddr":"10.1.10.100","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"iY3klKKVoLIGj3M39at6","level":3,"time":"2018-02-01T21:37:20+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"OwKulnUKLeLREJGmjy4l","level":3,"time":"2018-02-01T21:37:58+01:00","remoteAddr":"10.1.10.100","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"m15ciedffVE8Dd86IagM","level":3,"time":"2018-02-01T21:38:11+01:00","remoteAddr":"10.1.10.100","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"KzGYdnSPl8wjcEwcXXkx","level":3,"time":"2018-02-01T21:38:19+01:00","remoteAddr":"10.1.10.100","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"NahFXmFj96NQea3dVBq8","level":3,"time":"2018-02-01T21:38:24+01:00","remoteAddr":"177.228.173.217","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"5RYTK7MN7mL6d9To7Sq2","level":3,"time":"2018-02-01T21:38:48+01:00","remoteAddr":"188.194.5.243","user":"Username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"AuF1FlaqSCYCsFHh5Wo3","level":3,"time":"2018-02-01T21:39:49+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"Ru5ErVXOFEH5RCJKJ2mi","level":3,"time":"2018-02-01T21:39:57+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"h9uQkWJjIanikBurEpMq","level":3,"time":"2018-02-01T21:40:06+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"FohX38maWThkqOUg0ZGM","level":3,"time":"2018-02-01T21:40:35+01:00","remoteAddr":"177.228.173.217","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"CXTephT2yYFo163s5I2o","level":3,"time":"2018-02-01T21:44:11+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"bin3Ur75AADGhbDwCUnE","level":3,"time":"2018-02-01T21:44:16+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"aRhZ86SaZxPqlpdiEkzM","level":3,"time":"2018-02-01T21:45:01+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"xYTfKvhJwMcNe8qiJoDf","level":3,"time":"2018-02-01T21:46:41+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"ho9DbvBplsKfSXhAZHqS","level":3,"time":"2018-02-01T21:46:55+01:00","remoteAddr":"10.1.10.100","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"VMtKH7xQzexC1Ba0ZgN3","level":3,"time":"2018-02-01T21:47:20+01:00","remoteAddr":"10.1.10.100","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"e0La1WLPHIHXT4FpataT","level":3,"time":"2018-02-01T21:47:39+01:00","remoteAddr":"10.1.10.100","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"lTG7Wg50UmRDT0zfBF1c","level":3,"time":"2018-02-01T21:47:54+01:00","remoteAddr":"10.1.10.100","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"5he2vgSFCaeZsOceJwnx","level":3,"time":"2018-02-01T21:48:08+01:00","remoteAddr":"177.228.173.217","user":"username","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"D80hMglO0m5Iyt5npE6w","level":3,"time":"2018-02-01T21:48:13+01:00","remoteAddr":"10.1.10.100","user":"--","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"1bJUOgM9AlNhDTh2TmeA","level":3,"time":"2018-02-01T21:48:18+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"vDrf7DQwz9beT2vIWdEC","level":3,"time":"2018-02-01T21:50:44+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"pHxII53Fd2zY5TcdTqYB","level":3,"time":"2018-02-01T21:50:47+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"PHP","method":"GET","url":"/index.php/core/js/oc.js?v=d6d3c93f624677875cb1067a2716ba6b","message":"json_decode() expects parameter 1 to be string, array given at /var/www/owncloud/apps/files_reader/lib/Hooks.php#37"}
{"reqId":"tTNHH975U3wjN3MPWWGK","level":1,"time":"2018-02-01T21:51:05+01:00","remoteAddr":"10.1.10.100","user":"usernaMe","app":"files_reader","method":"POST","url":"/index.php/settings/ajax/disableapp.php","message":"user <carlos>, displayName <Carlos Damken> disabled app "files_reader" [CLIENT_IP: 10.1.10.100] [USER_AGENT: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0]"}
```

**Comment by Yetangitu** [comment]
Hm, strange, this might be yet another incompatibility between NC and OC.

Please post the value of `oc_appconfig`, you can find it by looking at the source for any OC-generated page (in the `<head>` section, look for a large block of JSON) or by requesting `https://your_domain.example.org/core/js/oc.js` (of course you need to replace _your_domain.example.org_ with your real domain and add any necessary path components to reach your OC instance). Don't post the whole thing, only post the value of `oc_appconfig`. For me it looks as follows:

```JSON
var oc_appconfig={"core":{"defaultExpireDateEnabled":false,"defaultExpireDate":null,"defaultExpireDateEnforced":null,"enforcePasswordForPublicLink":false,"enableLinkPasswordByDefault":false,"sharingDisabledForUser":false,"resharingAllowed":true,"remoteShareAllowed":true,"federatedCloudShareDoc":"https:\/\/docs.nextcloud.com\/server\/12\/go.php?to=user-sharing-federated","allowGroupSharing":true},"filesReader":{"enableEpub":"true","enablePdf":"true","enableCbx":"true"},"shareByMailEnabled":true,"shareByMail":{"enforcePasswordProtection":false}};
```

**Comment by Yetangitu** [comment]
_Nextcloud_ encodes `oc_appconfig` as JSON while _Owncloud_ does not (yet - it _does_ encode most other properties as JSON so I assume (yes, yes, I know) it'll end up encoding this as well).

Problem fixed in 1.2.2 by adding a test for JSON-encodedness, to be removed when _Owncloud_ has joined the fray.
